README.md

```diff
@@ -15,13 +15,15 @@ Community (college) maintained list of algorithm implementations and codes.
 
 ## Resources
 
-* [Popular Data Structures and Algorithms - Codechef](https://discuss.codechef.com/questions/48877/data-structures-and-algorithms)
-* [Intro to Algorithms - Khan Academy](https://www.khanacademy.org/computing/computer-science/algorithms)
 * [Algorithms - Learneroo](https://www.learneroo.com/subjects/8)
 * [Awesome-Algorithms](https://github.com/tayllan/awesome-algorithms)
+* [Algorithms List - GeekforGeeks](http://www.geeksforgeeks.org/fundamentals-of-algorithms/)
```

Comment:
It should be `GeeksForGeeks` . 
Also please change the PR title to reflect the change done in this PR.